packages/bitcore-payid/src/index.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  AddressDetailsType,
+  verifySignedAddress,
+} from '@payid-org/utils';
+import { JWS } from 'jose';
```

Comment:
jose? Do we really need that new dep? Can't we create/verify the signature with bitcore-lib?

Reply:
The jose lib is a pretty standard one across languages for cross-platform and cross-algorithm key conversion. I used it here for key conversion for the general case when someone might provide their own signing key and not one generated by bitcore-lib. However, it's not needed for verification which is what this MR pertains to and might even be over-kill for signing. I'm not 100% sure that it's needed just yet. Removed for now.

Reply:
I ended up leaving the jose dep in due to PayId's integral dependence on it. I'm still investigating ways to build the objects I need without relying on a whole package, though. You can see in the tests how I'm using it to transform a key to the JWK format that PayId expects.

As for creating and verifying with bitcore-lib, it's non-trivial if we want it to be interoperable with other PayId instances.